fvtest/gctest/gcTestHelpers.cpp

```diff
@@ -60,6 +60,8 @@ GCTestEnvironment::initParams()
 				params.push_back(line);
 			}
 			omrfile_close(fileDescriptor);
+		} else if (strncmp(_argv[i], "-keepVerboseLog", strlen("-keepVerboseLog")) == 0) {
```

Comment:
Why use strncmp instead of strcmp? Do you expect to accept options that have -keepVerboseLog as a prefix?